lib/utils/is-electron.ts

```diff
@@ -0,0 +1,9 @@
+// https://github.com/atom/electron/issues/22
+
+export const isElectron: () => boolean = () => {
+  debugger;
+  return !!window?.process?.type;
+};
```

Comment:
both of these exports are functions and they shouldn't need to be. let's simplify now. there was a reason they were IIFEs before, but I don't remember. maybe to lazy-evaluate the code? that wouldn't be a strong reason…

```ts
export const isElectron = !! window?.process?.type;

export const isMac = window?.process?.platofrm === 'darwin';
```